### PR TITLE
Use --key=value notation for CLI

### PR DESCRIPTION
**Pull request by VicDeo**
Fixes https://github.com/owncloud/updater/issues/383
To test try to start update from CLI on master/stable9.1/stable9

```
$ php updater/application.php upgrade:start
```
The result is `Symfony\Component\Process\Exception\ProcessFailedException`

### CODE REVIEW

File: src/Utils/OccRunner.php [hunk 1]
```diff
@@ -55,7 +55,7 @@ public function run($command, $args = [], $asJson = false){
 			$cmdLine = trim($command . ' ' . $extra);
 			foreach ($args as $optionTitle => $optionValue){
 				if (strpos($optionTitle, '--') === 0){
-					$line = trim("$optionTitle $optionValue");
+					$line = trim("$optionTitle=$optionValue");
```
Comment (PVince81) [inline src/Utils/OccRunner.php]: shouldn't these also be escaped somehow ?

Comment (VicDeo) [inline src/Utils/OccRunner.php]: @PVince yes, a few lines below. And the issue is caused by this escaping, that's why we need to get rid of space symbol here

**Comment by davitol** [comment]
:+1:

Solves the following warning error:

```
  [Symfony\Component\Process\Exception\ProcessFailedException]                                        
  The command "php /opt/owncloud/occ --no-warnings app:list --output=json '--shipped false'" failed.  
  Exit Code: 1(General error)                                                                         
  Output:                                                                                             
  ================                                                                                    
  Error Output:                                                                                       
  ================                                                                                    
                                                                                                      
    [Symfony\Component\Console\Exception\RuntimeException]                                            
    The "--shipped false" option does not exist.                                                      
                                                                                                      
  app:list [--output [OUTPUT]] [--shipped SHIPPED]
```

**Comment by VicDeo** [comment]
Confirmed by the original reporter in https://github.com/owncloud/updater/issues/383

File: src/Utils/OccRunner.php (hunk 1 above)
Comment (PVince81) [inline src/Utils/OccRunner.php]: Hmm, I was mostly thinking that the key and value need to be escaped separately. But if you think that the current way is fine, then okay.

**Comment by VicDeo** [comment]
Stable9.1: https://github.com/owncloud/updater/pull/394

**Comment by PVince81** [comment]
is this needed in 9.0 also ?

**Comment by davitol** [comment]
@PVince81 Yes, it is needed, it also fails in 9.0

**Pull request merged**